scripts/spartan-agent-setup.ps1

```diff
@@ -28,37 +28,19 @@ function New-Environment {
     New-Directory $SPARTAN_RELEASE_DIR
     New-Directory $SPARTAN_SERVICE_DIR
     New-Directory $SPARTAN_LOG_DIR
-    $spartanReleaseZip = Join-Path $env:TEMP "spartan-release.zip"
-    Write-Output "Downloading latest Spartan build"
-    Start-ExecuteWithRetry { Invoke-WebRequest -UseBasicParsing -Uri $SPARTAN_LATEST_RELEASE_URL -OutFile $spartanReleaseZip }
-    Write-Output "Extracting Spartan zip archive to $SPARTAN_RELEASE_DIR"
-    Expand-Archive -LiteralPath $spartanReleaseZip -DestinationPath $SPARTAN_RELEASE_DIR
+    $spartanReleaseZip = Join-Path $AGENT_BLOB_DEST_DIR "spartan-release.zip"
```

Comment:
Zip from the blob archive is called `spartan.zip`

Reply:
Will address